### DISCUSSION TITLE
Using selector with transform:scale() in sprinkles API

### DISCUSSION THREAD

**Discussion by adamlbunch** [vanilla-extract-css/vanilla-extract#523] [Q&A] [answered]
Hi, I'm trying to do this with sprinkles:

```
const selectorProperties = defineProperties({
  conditions: {
    default: {},
    hover: { selector: '&:hover' },
  },
  defaultCondition: 'default',
  properties: {
    transform: ['scale(1.1)'],
  },
});
```

But I'm getting this error:

```
ERROR in ./src/components/input-field/input-field.css.ts
Module build failed (from ./node_modules/@vanilla-extract/webpack-plugin/loader/dist/vanilla-extract-webpack-plugin-loader.cjs.js):
Error: Invalid selector: sprinkles_transform_scale(1.1)_hover__1u37gwpaw:hover
```

Any clue what might be happening? I should be able to use sprinkles to restrict styling under the hover condition right? Thank you!

**Comment by mattcompiles** [comment, marked as answer]
Hey Bud 👋 

Just confirming you're on the latest version of Sprinkles? We fixed a similar error last week. If that's not it then could you create a small reproduction and create a new issue? Thanks.

**Reply by adamlbunch** [reply]
Updating worked! My apologies, I installed 2 weeks ago so just assumed that I was doing something wrong. Thanks!
